### ISSUE TITLE
New Spinner causes significant output to CI/CD Pipeline Jobs with gitlab

### ISSUE THREAD

**Issue by jlaule** [redaxmedia/handpick#24]
When the gitlab CI/CD Pipeline runs to build our images using handpick the spinner information continually gets appended and not replaced.

Small snippet:
```
 ---> Running in 8017071ba57e
[?25l[2K[1G⠋ Hand picking DIRTY dependencies and devDependencies and devDsiDependencies via NPM[?25l[2K[1G⠙ Hand picking DIRTY dependencies and devDependencies and devDsiDependencies via NPM[?25l[2K[1G⠹ Hand picking DIRTY dependencies and devDependencies and devDsiDependencies via NPM[?25l[2K[1G⠸ Hand picking DIRTY dependencies and devDependencies and devDsiDependencies via NPM[?25l[2K[1G⠼ Hand picking DIRTY dependencies and devDependencies and devDsiDependencies via NPM[?25l[2K[1G⠴ Hand picking DIRTY dependencies and devDependencies and devDsiDependencies via NPM[?25l[2K[1G⠦ Hand picking DIRTY dependencies and devDependencies and devDsiDependencies via NPM[?25l[2K[1G⠧ Hand picking DIRTY dependencies and devDependencies and devDsiDependencies via NPM[?25l[2K[1G⠇ Hand picking DIRTY dependencies and devDependencies and devDsiDependencies via NPM[?25l[2K[1G⠏ Hand picking DIRTY dependencies and devDependencies and devDsiDependencies via NPM[?25l[2K[1G⠋ Hand picking DIRTY dependencies and devDependencies and devDsiDependencies via NPM[?25l[2K[1G⠙ Hand picking DIRTY dependencies and devDependencies and devDsiDependencies via NPM[?25l[2K[1G⠹ Hand picking DIRTY dependencies and devDependencies and devDsiDependencies via NPM[?25l[2K[1G⠸ Hand picking DIRTY dependencies and devDependencies and devDsiDependencies via NPM[?25l[2K[1G⠼ Hand picking DIRTY dependencies and devDependencies and devDsiDependencies via NPM[?25l[2K[1G⠴ Hand picking DIRTY dependencies and devDependencies and devDsiDependencies via NPM[?25l[2K[1G⠦ Hand picking DIRTY dependencies and devDependencies and devDsiDependencies via NPM[?25l[2K[1G⠧ Hand picking DIRTY dependencies and devDependencies and devDsiDependencies via NPM[?25l[2K[1G⠇ Hand picking DIRTY dependencies and devDependencies and devDsiDependencies via NPM[?25l[2K[1G⠏ Hand picking DIRTY dependencies and devDependencies and devDsiDependencies via NPM[?25l[2K[1G⠋ Hand picking DIRTY dependencies and devDependencies and devDsiDependencies via NPM[?25l[2K[1G⠙ Hand picking DIRTY dependencies and devDependencies and devDsiDependencies via NPM[?25l[2K[1G⠹ Hand picking DIRTY dependencies and devDependencies and devDsiDependencies via NPM[?25l[2K[1G⠸ Hand picking DIRTY dependencies and devDependencies and devDsiDependencies via NPM[?25l[2K[1G⠼ Hand picking DIRTY dependencies and devDependencies and devDsiDependencies via NPM[?25l[2K[1G⠴ Hand picking DIRTY dependencies and devDependencies and devDsiDependencies via NPM[?25l[2K[1G⠦ Hand picking DIRTY dependencies and devDependencies and devDsiDependencies via NPM[?25l[2K[1G⠧ Hand picking DIRTY dependencies and devDependencies and devDsiDependencies via NPM[?25l[2K[1G⠇ Hand picking DIRTY dependencies and devDependencies and devDsiDependencies via NPM[?25l[2K[1G⠏ Hand picking DIRTY dependencies and devDependencies and devDsiDependencies via NPM[?25l[2K[1G⠋ Hand picking DIRTY dependencies and devDependencies and devDsiDependencies via NPM[?25l[2K[1G⠙ Hand picking DIRTY dependencies and devDependencies and devDsiDependencies via NPM[?25l[2K[1G⠹ Hand picking DIRTY dependencies and devDependencies and devDsiDependencies via NPM[?25l[2K[1G⠸ Hand picking DIRTY dependencies and devDependencies and devDsiDependencies via NPM[?25l[2K[1G⠼ Hand picking DIRTY dependencies and devDependencies and devDsiDependencies via NPM[?25l[2K[1G⠴ Hand picking DIRTY dependencies and devDependencies and devDsiDependencies via NPM[?25l[2K[1G⠦ Hand picking DIRTY dependencies and devDependencies and devDsiDependencies via NPM[?25l[2K[1G⠧ Hand picking DIRTY dependencies and devDependencies and devDsiDependencies via NPM[?25l[2K[1G⠇ Hand picking DIRTY dependencies and devDependencies and devDsiDependencies via NPM[?25l[2K[1G⠏ Hand picking DIRTY dependencies and devDependencies and devDsiDependencies via NPM[?25l[2K[1G⠋ Hand picking DIRTY dependencies and devDependencies and devDsiDependencies via NPM[?25l[2K[1G⠙ Hand picking DIRTY dependencies and devDependencies and devDsiDependencies via NPM[?25l[2K[1G⠹ Hand picking DIRTY dependencies and devDependencies and devDsiDependencies via NPM[?25l[2K[1G⠸ Hand picking DIRTY dependencies and devDependencies and devDsiDependencies via NPM[?25l[2K[1G⠼ Hand picking DIRTY dependencies and devDependencies and devDsiDependencies via NPM[?25l[2K[1G⠴ Hand picking DIRTY dependencies and devDependencies and devDsiDependencies via NPM[?25l[2K[1G⠦ Hand picking DIRTY dependencies and devDependencies and devDsiDependencies via NPM[?25l[2K[1G⠧ Hand picking DIRTY dependencies and devDependencies and devDsiDependencies via NPM[?25l[2K[1G⠇ Hand picking DIRTY dependencies and devDependencies and devDsiDependencies via NPM[?25l[2K[1G⠏ Hand picking DIRTY dependencies and devDependencies and devDsiDependencies via NPM[?25l[2K[1G⠋ Hand picking DIRTY dependencies and devDependencies and devDsiDependencies via NPM[?25l[2K[1G⠙ Hand picking DIRTY dependencies and devDependencies and devDsiDependencies via NPM[?25l[2K[1G⠹ Hand picking DIRTY dependencies and devDependencies and devDsiDependencies via NPM[?25l[2K[1G⠸ Hand picking DIRTY dependencies and devDependencies and devDsiDependencies via NPM[?25l[2K[1G⠼ Hand picking DIRTY dependencies and devDependencies and devDsiDependencies via NPM[?25l[2K[1G⠴ Hand picking DIRTY dependencies and devDependencies and devDsiDependencies via NPM[?25l[2K[1G⠦ Hand picking DIRTY dependencies and devDependencies and devDsiDependencies via NPM[?25l[2K[1G⠧ Hand picking DIRTY dependencies and devDependencies and devDsiDependencies via NPM[?25l[2K[1G⠇ Hand picking DIRTY dependencies and devDependencies and devDsiDependencies via NPM[?25l[2K[1G⠏ Hand picking DIRTY dependencies and devDependencies and devDsiDependencies via NPM[?25l[2K[1G⠋ Hand picking DIRTY dependencies and devDependencies and devDsiDependencies via NPM[?25l[2K[1G⠙ Hand picking DIRTY dependencies and devDependencies and devDsiDependencies via NPM[?25l[2K[1G⠹ Hand picking DIRTY dependencies and devDependencies and devDsiDependencies via NPM[?25l[2K[1G⠸ Hand picking DIRTY dependencies and devDependencies and devDsiDependencies via 
```

**Comment by redaxmedia** [comment]
I just released handpick@5.0.2 that includes a isTTY (is interactive terminal) and unicode check for the spinner. I don't plan to provide fallbacks at this moment just in case someone is reading this.

@jlaule Please give it a try, I had no time to test in inside a real CI but it should work.

**Comment by jlaule** [comment]
Yes that appears to fix the issue with CI/CD in gitlab.
